asyncpraw/endpoints.py

```diff
@@ -0,0 +1,209 @@
+"""List of API endpoints PRAW knows about."""
+
+# flake8: noqa
+# fmt: off
+API_PATH = {
+    "about_edited":            "r/{subreddit}/about/edited/",
+    "about_log":               "r/{subreddit}/about/log/",
```

Comment:
@LilSpazJoekp I had meant that such objects could have been imported from praw.
On one hand it adds to the complexity. On the positive side, you would not have to maintain this yourself

Reply:
I don't think it would be hard to maintain. I would apply patches from praw whenever changes were made to it.

Reply:
One problem can occur whenever API endpoints are modified by PRAW and the changed methods are not backported to asyncpraw. Also models have mixins which have methods that would need to be async-ified.

Reply:
Exactly what I was thinking. So I think it would just be easier to do it manually each commit.